src/Uno.Templates/content/unoapp/.vscode/launch.json

```diff
@@ -41,17 +41,17 @@
       // Use IntelliSense to find out which attributes exist for C# debugging
       // Use hover for the description of the existing attributes
       // For further information visit https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger-launchjson.md
-      "name": "Skia.GTK (Debug)",
+      "name": "Uno Desktop (Debug)",
```

Comment:
That's under `#if (useGtk)` so it won't be added to desktop projects.
OTOH it might not be needed as the extension knows (PR pending) about the new TFM.